<launch>
	<include file="$(find cwru_configs)/$(optenv ROBOT sim)/base/static_transform.launch" />
    <node pkg="cwru_base" type="crio_receiver" name="crio_receiver" output="screen">
	    <rosparam command="load" file="$(find cwru_configs)/$(optenv ROBOT sim)/base/diagnostics.yaml" />
	    <rosparam command="load" file="$(find cwru_configs)/$(optenv ROBOT sim)/base/base.yaml" />
    </node>
    <node pkg="cwru_base" type="odom_translator.py" name="odom_translator">
        <rosparam command="load" file="$(find cwru_configs)/$(optenv ROBOT sim)/base/base.yaml" />
    </node>
    
    <node pkg="cwru_base" type="twist_receiver.py" name="twist_receiver">
        <rosparam command="load" file="$(find cwru_configs)/$(optenv ROBOT sim)/base/base.yaml" />
    </node>

    <include file="$(find cwru_configs)/$(optenv ROBOT sim)/base/sonar.launch" />
</launch>
